pkg/cluster/cache_whitebox_test.go

```diff
@@ -206,32 +199,27 @@ func TestGetClustersByType(t *testing.T) {
 		defer resetClusterCache()
 
 		// noise
-		host := newTestCachedToolchainCluster(t, "cluster-host", Host, ready)
+		host := newTestCachedToolchainCluster(t, "cluster-host", ready)
 		clusterCache.addCachedToolchainCluster(host)
-		member1 := newTestCachedToolchainCluster(t, "cluster-1", Member, ready)
+		member1 := newTestCachedToolchainCluster(t, "cluster-1", ready)
 		clusterCache.addCachedToolchainCluster(member1)
-		member2 := newTestCachedToolchainCluster(t, "cluster-2", Member, ready)
+		member2 := newTestCachedToolchainCluster(t, "cluster-2", ready)
 		clusterCache.addCachedToolchainCluster(member2)
-		member3 := newTestCachedToolchainCluster(t, "cluster-3", Member, notReady)
+		member3 := newTestCachedToolchainCluster(t, "cluster-3", notReady)
 		clusterCache.addCachedToolchainCluster(member3)
 
 		t.Run("get only ready member clusters", func(t *testing.T) {
 			//when
 			clusters := GetMemberClusters(Ready)
 
 			//then
-			assert.Len(t, clusters, 2)
```

Comment:
here it will be 3 bcoz, the function getmemberclusters and get host cluster is same it gets all the cluster, based on status here it is giving ready status hence the len will be 3 i shall add it